commit 24f48b86c8988ee3aaebc5f303d71e9d789f77b6
Author: Jan Engelhardt <jengelh@inai.de>
Date:   Thu Apr 17 16:33:18 2014 +0200

    dbd_sqlite3: resolve a stack buffer overflow
    
    This statement as such has no effect and is responsible for
    subsequent invocation of undefined behavior when strncpy writes to a
    potentially too small buffer. The intention here was likely to define
    a new "word_lower" that is big enough (and shadows the old one).

diff --git a/drivers/sqlite3/dbd_sqlite3.c b/drivers/sqlite3/dbd_sqlite3.c
index 6d3c0c3..67a782e 100644
--- a/drivers/sqlite3/dbd_sqlite3.c
+++ b/drivers/sqlite3/dbd_sqlite3.c
@@ -1451,7 +1451,7 @@ static int getTables(char** tables, int
 		    break;
 		  }
 
-		  word_lower[item-start+1];
+		  char word_lower[item-start+1];
 		  strncpy(word_lower,start,item-start);
 		  word_lower[item-start] = '\0';
 		  int i = 0;
